ledger/src/shred.rs

```diff
@@ -403,7 +403,10 @@ impl Shred {
                 .for_each(|(accum, seed)| *accum ^= *seed)
         });
         // convert accum into a u16
-        ((accum[0] as u16) << 8) | accum[1] as u16
+        let version = ((accum[0] as u16) << 8) | accum[1] as u16
+
+        // ensure version is never zero, to avoid looking like an uninitialized version
+        version.saturating_add(1)
```

Comment:
No test 😱 
Beyond a loop that generates hundreds of thousands of hashes I don't know how to generate an input hash would produce a version of 0